src/main/java/org/jenkins/plugins/statistics/gatherer/listeners/QueueStatsListener.java

```diff
@@ -183,9 +182,7 @@ private String getRestUrl() {
 
     private QueueStats getCiQueue(Item item) {
         QueueStats queue = new QueueStats();
-        String ciUrl = Jenkins.getInstance() != null
-                ? Jenkins.getInstance().getRootUrl()
-                : "";
+        String ciUrl = Jenkins.getInstance().getRootUrl();
```

Comment:
You are removing a null check here, are you sure that is aways going to be notnull?

Reply:
`getInstance()` is annotated with `@Nonnull` since some time and throwing an `@IllegalStateException` if no instance is available. Also see https://github.com/jenkinsci/jenkins/pull/2297.